src/_locales/en/messages.json

```diff
@@ -240,7 +243,12 @@
     "settings_appearance_dark": "Dark",
     "settings_appearance_light": "Light",
     "settings_language": "Language",
-    "settings_choose_eth_network": "Choose Ethereum network",
+    "settings_choose_eth_network": "Choose Ethereum Network",
+    "settings_post_replacer_all_posts": "All Posts",
+    "settings_post_replacer_enhanced_posts": "Enhanced Posts",
+    "settings_post_replacer_encrypted_posts": "Encrypted Posts",
+    "settings_post_replacement_scope": "Post Replacement Scope",
+    "settings_post_replacement_scope_desc": "Maskbook will create a similar post and replace the original post with it.",
```

Comment:
Maskbook will render the post to replace the original one to provide some enhancement.